guides/common/modules/proc_using-the-host-details-tab.adoc

```diff
@@ -0,0 +1,31 @@
+[id="using-the-host-details-tab_{context}"]
+= Using the Host Details Tab
+
+In {Project}, you can view details of a host name in the *Details* tab.
+You can expand and collapse individual cards and all links.
+Your browser remembers the card expansion and collapse state.
+
+.Procedure
+. In the {Project} web UI, navigate to *Hosts* > *All Hosts*.
+. Click the name of the host you want to view.
+. Select the *Details* tab.
+
+The cards in the *Details* tab show details for the *System properties*, *BIOS*, *Networking interfaces*, *Operating system*, *Provisioning templates*, and *Provisioning*.
+Registered content hosts show additional cards for *Registration details* and *Installed products*.
```

Comment:
Is this is Katello/Satellite/orcharhino only?

Reply:
@vsedmik can you confirm if this is Katello/Satellite/Orcharino only?

Reply:
Not sure about Orcharino, but it is Katello/Satellite.

Reply:
@parthaa Do you know if this would also apply to Orcharino or is it just Katello/Satellite?

Reply:
Confirmed with Partha to change it to Katello/Satellite for this section.